packages/schematics/angular/application/files/common-files/src/app/app.component.html.template

```diff
@@ -142,13 +142,13 @@
   .pill-group .pill:nth-child(6n + 2) {
     --pill-accent: var(--french-violet);
   }
-  .pill-group .pill:nth-child(6n + 3), 
+  .pill-group .pill:nth-child(6n + 3),
   .pill-group .pill:nth-child(6n + 4),
   .pill-group .pill:nth-child(6n + 5) {
     --pill-accent: var(--hot-red);
   }
 
-  .open-in-new-icon {
+  .pill-group svg {
     margin-inline-start: 0.25rem;
     font-size: 1rem;
```

Comment:
I think this now needs to be smaller, the icons are too large.

Reply:
![Uploading Screenshot 2023-11-07 at 16.31.30.png…]()

Reply:
<img width="767" alt="Screenshot 2023-11-07 at 16 31 30" src="https://github.com/angular/angular-cli/assets/17563226/fe875da5-5bd4-42d3-b3f0-c42bc9c2f53c">